bessctl/conf/testing/module_tests/mlfqueue.py

```diff
@@ -0,0 +1,36 @@
+import sugar
+from module import *
+from port import *
+
+def basic_throughput_test():
```

Comment:
It's really an output test -- that the thing outputs the packets you expect it to. A throughput test would be benchmarking packets per second. Rename?